docs/source/getting_started/01-getting-started.md

```diff
@@ -58,13 +58,12 @@ It's up to the implementer to populate the `DiffSync`'s internal cache with the
 
 ## Model Processing Ordering Logic
 
-The models will be processed in a specfic order as defined by `top_level` atttribute on the `DiffSync` object and then the `_childres` attribute on the `DiffSyncModel`. The processing algorithm can be described as:
+The models will be processed in a specfic order as defined by `top_level` atttribute on the `DiffSync` object and then the `_children` attribute on the `DiffSyncModel`. The processing algorithm is technically a "Preorder Tree Traversal", which means that "a parent node is processed before any of its child nodes is done." This can be described as:
```

Comment:
Who are you quoting here when you quote "a parent node is [...]"?

Reply:
wikipedia.

Reply:
I guess this is largely personal taste but I would drop the quotes here. Feel free to do that or to not do it.